docs/authoring.md

```diff
@@ -108,7 +109,7 @@ Follow these steps to add an application manifest file to your project:
 
 1. In Visual Studio, right click on the project node under **Solution Explorer** and click **Add -> New Item**. Search for the **Text File** template and name your file `YourNativeApp.exe.manifest`.
 
-2. Add your activatable class registrations to the manifest file. Below is an example with a C#/WinRT component `MyAuthoredComponent` and activatable classes `WinRTComponent.Class*`. Note that `WinRT.Host.dll` must be specified as the file name.
+2. Add your activatable class registrations to the manifest file. Below is an example with a C#/WinRT component `MyAuthoredComponent` and activatable classes `WinRTComponent.Class*`. Note that `WinRT.Host.dll` must be specified as the file name, without a relative path.
 
       ```xml

```

Comment:
btw / for future reference that this is automatable if the types exist in a winmd (see the MT task I have in the targets file for the nuget package called `Unpackaged`)

Reply:
see https://github.com/asklar/xaml-islands/blob/main/build/Unpackaged.targets#L8-L38

Reply:
great!! I think that would help us with https://github.com/microsoft/CsWinRT/issues/789